src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall());
+
```

Comment:
you don't need three instances of `Lottery` class - you can have just one instance and use a `for` loop

```suggestion
        Lottery lottery = new Lottery();
        for(int i = 0; i < NUMBER_OF_BALLS; i++ {
           System.out.println(lottery.getRandomBall());
        }

```
`NUMBER_OF_BALLS` - is a constant that you need to create:
```
private static final int NUMBER_OF_BALLS = 3;
```